### PR TITLE
Add variations on 'sheck->check'

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
please review

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -16422,6 +16422,10 @@ shashes->slashes
 shatow->château
 shbang->shebang
 sheat->sheath, sheet, cheat,
+sheck->check
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: Also "cheque" (BE)

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Added in 0ae0633

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -16436,6 +16436,11 @@ shashes->slashes
 shatow->château
 shbang->shebang
 sheat->sheath, sheet, cheat,
+sheck->check, cheque,
+shecked->checked
+shecks->checks, cheques,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Possibly suggest shucks

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: So...  
`sheck->check, cheque, shuck`
`shecks->checks, cheques, shucks`  
?

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @peternewman please confirm :point_up_2:

**Pull request merged**